options/options.c

```diff
@@ -891,6 +892,10 @@ static const m_option_t mp_opts[] = {
     {"", OPT_SUBSTRUCT(opengl_opts, opengl_conf)},
 #endif
 
+#if HAVE_EGL || HAVE_EGL_ANDROID || HAVE_EGL_ANGLE_WIN32
+    {"", OPT_SUBSTRUCT(egl_opts, egl_conf)},
```

Comment:
could use option prefix here

Reply:
modified in a09f2d2d38b1f4395b4815661fc4af40b993d54b